PurchasesTests/Caching/DeviceCacheTests.swift

```diff
@@ -385,19 +392,19 @@ class DeviceCacheTests: XCTestCase {
         let key = "some key"
         let value = "some value"
         let setTime = NSDate()
-        let isSynced = true
+        let isSynced: NSNumber = true
```

Comment:
This would makes more sense as `Bool`. What required this change?

Reply:
Well, I tried to silence this warning 
![Screenshot 2022-01-05 at 09 11 50](https://user-images.githubusercontent.com/1409041/148182997-2e0f3150-d18a-4e99-8b1e-7788d76f684a.png)

But, I found another way to do it, just using the `init(value: Bool)` constructor instead 👍🏼